scripts/hudson/narayana.sh

```diff
@@ -628,10 +628,13 @@ EOT
     rm -rf narayana-tomcat
     git clone https://github.com/jbosstm/narayana-tomcat.git
     echo "Executing Narayana Tomcat tests"
-    ./build.sh -f narayana-tomcat/pom.xml -fae -B -P${ARQ_PROF}-tomcat ${CODE_COVERAGE_ARGS} -Dtest.db.type=h2 -Dversion.org.jboss.narayana=${NARAYANA_CURRENT_VERSION} "$@" ${IPV6_OPTS} clean install "$@"
+    grep "<version.org.jboss.narayana>5.9.11.Final</version.org.jboss.narayana>" narayana-tomcat/pom.xml
```

Comment:
Seems a bit brittle hardcoding reliance on 5.9.11.Final but I guess we will quickly see the problem if the version in the narayana-tomcat changes.

Reply:
+1, but I should have said that somewhere maybe - i.e. when the version in our fork of narayana-tomcat's pom.xml changes, to update this. I am not sure if hiding the comment above here helps in isolation (but maybe it would help when debugging) but I guess we might try to find somewhere else (maybe a second comment in our fork of narayana-tomcat's pom.xml?)

I will add a comment to this file though

Reply:
I did a little more than the comment, hopefully the changes in this repo are clearer.